mail/emailrelay/files/emailrelay.config

```diff
@@ -0,0 +1,27 @@
+config emailrelay 'server'
+	option enabled '0'
+	option mode 'server'
+	option port '25'
+	option remote_clients '0'
+#	option server_tls '/etc/path/to/certificate'
+#	option server_auth '/etc/emailrelay.auth'
+#	option extra_cmdline ''
+
+config emailrelay 'proxy'
+	option enabled '0'
+	option mode 'proxy'
+	option smarthost '192.0.2.1:25'
+	option port '25'
+	option remote_clients '0'
+#	option server_tls '/etc/path/to/certificate'
+#	option server_auth '/etc/emailrelay.auth'
+#	option client_tls '1'
+#	option client_auth '/etc/emailrelay.auth'
+#	option extra_cmdline ''
+
+config emailrelay 'cmdline'
+	option enabled '0'
+	option mode 'cmdline'
+	# specify all arguments that should be passed to emailrelay here
```

Comment:
Please just add a comment with the url for cmdline reference, e.g.
#see http://emailrelay.sourceforge.net/reference.html for command line reference